src/SFA.DAS.AssessorService.Web/Views/Application/StandardApplications.cshtml

```diff
@@ -26,23 +29,24 @@
                     {
                         if (!string.IsNullOrEmpty(application.ApplyData?.Apply?.StandardName))
                         {
-                        <tr>
-                            <th class="govuk-table__header" scope="row">
-                                @application.ApplyData?.Apply?.StandardName
-                            </th>
-                            <!--<td class="govuk-table__cell">@application.ApplicationId</td>-->
-                            <td class="govuk-table__cell govuk-table__cell--numeric">@application.ApplicationStatus</td>
-                            <td class="govuk-table__cell govuk-table__cell--numeric">
-                                <a class="govuk-link" href="@Url.Action("SequenceSignPost", "Application", new { Id = application.Id })">View</a>
-                            </td>
-                        </tr>
+                            <tr>
+                                <th class="govuk-table__header" scope="row">
+                                    @application.ApplyData?.Apply?.StandardName
+                                    @if (application.ApplyData?.Apply?.Versions != null && application.ApplyData?.Apply?.Versions.Count > 0)
+                                    {
+                                        <p class="govuk-body">Version @application.ApplyData?.Apply?.Versions[0].</p>
```

Comment:
Just trying to find the page for this to match to the prototype, if there is more than one version, do we not want to list the versions rather than show the latest?

Reply:
Changed to list the versions